src/test/java/io/reactivex/internal/operators/flowable/FlowableRetryWithPredicateTest.java

```diff
@@ -466,35 +467,40 @@ public boolean test(Integer n, Throwable e) throws Exception {
 
     @Test
     public void retryBiPredicateDisposeRace() {
-        for (int i = 0; i < TestHelper.RACE_DEFAULT_LOOPS; i++) {
-            final PublishProcessor<Integer> pp = PublishProcessor.create();
+        RxJavaPlugins.setErrorHandler(Functions.emptyConsumer());
```

Comment:
hm, did it flake?

Reply:
One valid outcome was an error that was flooding the console. It is not interesting here so itbis suppressed.